setup.py

```diff
@@ -67,7 +67,7 @@ def _get_dbt_core_version():
     include_package_data=True,
     install_requires=[
         "dbt-core~={}".format(dbt_core_version),
-        "snowflake-connector-python[secure-local-storage]>=2.4.1,<2.8.2",
+        "snowflake-connector-python[secure-local-storage]==2.9.0",
```

Comment:
We haven't landed on an exact range for this yet.

This is the current widest range known to work on across all OSes we test against in CI:
```suggestion
        "snowflake-connector-python[secure-local-storage]>=2.4.1,<2.9.0",
```

Reply:
It's worth pointing out that this passes the automated checks. However, our automation does not run the integration tests if only the requirements files (`requirements-dev.txt` and `setup.py`) are updated. I believe this is a gap in our automation as we would probably want the opposite (all integration tests in the matrix run). If you manually run the integration tests on this branch with 2.9.0 in there, it will fail.